crypto/src/main/java/org/cryptimeleon/incentive/crypto/proof/spend/tree/SpendDeductLeafNode.java

```diff
@@ -22,13 +20,4 @@ public SpendDeductLeafNode(String leafName) {
     public String getLeafName() {
         return leafName;
     }
-
-    /**
-     * Function that determines whether this leaf node's ZKP is satisfied by the given old and new points vectors.
-     *
-     * @param pointsVector    vector containing the old token's points
-     * @param newPointsVector vector containing the points the users want for their new token
-     * @return whether the point vectors satisfy the ZKP relation
-     */
-    public abstract boolean isValidForPoints(RingElementVector pointsVector, RingElementVector newPointsVector);
 }
```

Comment:
How is determined whether a leaf node's ZKP is satisfied with the passed point vector?

Reply:
Basically implement this method matching the semantics of the corresponding ZKP